packages/rum/src/domain/getSessionReplayLink.spec.ts

```diff
@@ -0,0 +1,83 @@
+import type { RumConfiguration, ViewContexts } from '@datadog/browser-rum-core'
+import { createRumSessionManagerMock } from '../../../rum-core/test'
+import { getSessionReplayLink } from './getSessionReplayLink'
+import { addRecord, resetReplayStats } from './replayStats'
+
+const DEFAULT_CONFIGURATION = {
+  site: 'datad0g.com',
+} as RumConfiguration
+
+describe('getReplayLink', () => {
+  afterEach(() => {
+    resetReplayStats()
+  })
+  it('should return url without query param if no view', () => {
+    const sessionManager = createRumSessionManagerMock().setId('session-id-1')
+    const viewContexts = { findView: () => undefined } as ViewContexts
+
+    const link = getSessionReplayLink(DEFAULT_CONFIGURATION, sessionManager, viewContexts)
+
+    expect(link).toBe('https://dd.datad0g.com/rum/replay/sessions/session-id-1')
+  })
+
+  const parameters: Array<[string, string | undefined, string]> = [
+    ['datadoghq.com', undefined, 'app.datadoghq.com'],
+    ['datadoghq.com', 'toto', 'toto.datadoghq.com'],
+    ['datad0g.com', undefined, 'dd.datad0g.com'],
+    ['datad0g.com', 'toto', 'toto.datad0g.com'],
+    ['us3.datadoghq.com', undefined, 'us3.datadoghq.com'],
+    ['us3.datadoghq.com', 'toto', 'toto.us3.datadoghq.com'],
+    ['us5.datadoghq.com', undefined, 'us5.datadoghq.com'],
+    ['us5.datadoghq.com', 'toto', 'toto.us5.datadoghq.com'],
+  ]
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏here too you might not need to test for every site/subdomain cases